python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -143,7 +143,7 @@ class ForcedPhotCcdConfig(ForcedPhotImageConfig):
     doApplyUberCal = lsst.pex.config.Field(
         dtype=bool,
         doc="Apply meas_mosaic ubercal results to input calexps?",
-        default=False
+        default=True
```

Comment:
Perhaps I wasn't in my right mind or I didn't understand the import of what I was recommending.

Let's make `doApplyUberCal=False` the default for the stack as a whole, and activate it in obs_subaru only.  We can revisit this choice once @parejkoj and friends are done with jointcal.